src/frontend/components/NewPlan.tsx

```diff
@@ -121,9 +104,36 @@ const MapView: React.FC<MapViewProps> = ({ plans, isLoaded, loadError }) => {
     if (!isLoaded || !mapRef.current) return;
 
     // Clear existing markers
-    markersRef.current.forEach(marker => marker.map = null);
+    markersRef.current.forEach(marker => marker.setMap(null));
     markersRef.current = [];
 
+    // Show selected destinations
```

Comment:
Looks great! Can have it as the base layer of the map view. Was thinking the map view might be helpful for displaying and comparing the three plans, showing their routes with markers.